controllers/controller.js

```diff
@@ -0,0 +1,12 @@
+const categories = require('../db/data/test-data/categories.js');
+const {selectCategories} = require('../models/model.js')
+
+
+exports.getCategories = (req, res) => {
+    selectCategories().then((Categories) => {
+      res.send({ Categories });
```

Comment:
Capitalized names are reserved for classes - we know this is an array of categories, so make sure to change these to lowercase as it's not following best JS practice